packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/customPropertyGridComponent.tsx

```diff
@@ -131,6 +132,20 @@ export class CustomPropertyGridComponent extends React.Component<ICustomProperty
                 );
             case InspectableType.Tab:
                 return <TextLineComponent key={inspectable.label} label={inspectable.label} value={" "} />;
+            case InspectableType.FileButton:
+                return (
+                    <FileButtonLineComponent
+                        key={inspectable.label}
+                        label={inspectable.label}
+                        onClick={
+                            inspectable.callback ||
+                            function () {
+                                console.warn("no call back function added");

```

Comment:
```suggestion
                                Tools.Warn("no callback function added");
```
We do our best not to use console (and especially console.warn) directly. We have Warn in Tools, making it possible to abstract Logging in different scenarios.